platform-tests/src/test/java/org/junit/platform/engine/support/store/NamespacedHierarchicalStoreTests.java

```diff
@@ -8,40 +8,43 @@
  * https://www.eclipse.org/legal/epl-v20.html
  */
 
-package org.junit.jupiter.engine.execution;
+package org.junit.platform.engine.support.store;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.platform.commons.test.ConcurrencyTestingUtils.executeConcurrently;
+import static org.mockito.Mockito.inOrder;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
 
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Function;
 
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
-import org.junit.jupiter.api.extension.ExtensionContextException;
 
 /**
- * Unit tests for {@link ExtensionValuesStore}.
+ * Unit tests for {@link NamespacedHierarchicalStore}.
  *
  * @since 5.0
- * @see org.junit.jupiter.engine.descriptor.ExtensionContextTests
  */
-public class ExtensionValuesStoreTests {
+public class NamespacedHierarchicalStoreTests {
```

Comment:
Check if some more tests should be moved from `rg.junit.jupiter.engine.execution.ExtensionContextStoreTests` to here such as `getOrComputeIfAbsentWithFailingCreator`.

Reply:
Added in 4d00cc38d0f12f1c69386708e37309def8c1e839.